src/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -3,20 +3,21 @@
     <ProjectGuid>{7061832A-E8CF-4AB6-A8DC-44D2F5A43A13}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ExternallyShipping>false</ExternallyShipping>
+    <NoWarn>436</NoWarn>
```

Comment:
What is this for?

Reply:
In order to disable CS0436 warnings:
Test project includes `src\Common\src\Interop` files as well and then its "Interop" class conflicts with imported "Interop" in System.Reflection.Metadata.

Reply:
Maybe use an alias?

Reply:
Can you share the full warning output without this? That might help to come up with alternatives. I would prefer to change the code to avoid the warnings or failing that, #pragma warning disable on specific places. Putting a NoWarn in the .csproj can mask future mistakes.

Reply:
Here are the warnings, there are 3:

```
D:\corefx\src\Common\src\CoreLib\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs(13,20): error CS0436: The type 'Interop' in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs' conflicts with the imported type 'Interop' in 'System.Reflection.Metadata, Version=1.4.4.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Using the type defined in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs'. [D:\corefx\src\System.Reflection.Metadata\tests\System.Reflection.Metadata.Tests.csproj]

TestUtilities\LoaderUtilities.cs(21,62): error CS0436: The type 'Interop' in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs' conflicts with the imported type 'Interop' in 'System.Reflection.Metadata, Version=1.4.4.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Using the type defined in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs'. [D:\corefx\src\System.Reflection.Metadata\tests\System.Reflection.Metadata.Tests.csproj]

TestUtilities\LoaderUtilities.cs(23,55): error CS0436: The type 'Interop' in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs' conflicts with the imported type 'Interop' in 'System.Reflection.Metadata, Version=1.4.4.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Using the type defined in 'D:\corefx\src\System.Reflection.Metadata\tests\..\..\Common\src\Interop\Windows\Interop.Libraries.cs'. [D:\corefx\src\System.Reflection.Metadata\tests\System.Reflection.Metadata.Tests.csproj]
```

With alias there's still reference to duplicated class name 'Interop.Kernel32' and when it comes to using #pragma option there's common file that wouldn't be alright to edit (`corefx\src\Common\src\CoreLib\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs`).

Would it be acceptable for test projects to have their own DllImports?

Reply:
This is because System.Reflection.Metadata is using InternalsVisibleTo.  Grrr :(